### PR TITLE
navigation bar added using hamburger

### PR DESCRIPTION
**Pull request by pallaviii21**
#39 
https://github.com/user-attachments/assets/5b1fc46b-a7e7-47ca-994c-c7b85ae0555f

//added a side navigation bar so that nav bar can be accessed  if the screen is minimized

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (Harshdev098) [inline .vscode/settings.json:2]: don't add this file in the remote repo

**Review by Harshdev098 — CHANGES_REQUESTED** [review]
Update the font size of the hamburger, also make the content of it taking full width and add padding, fontsize to the content in it because its not navigatible if someone clicks outside of the text layout

https://github.com/user-attachments/assets/18ce65eb-cf15-4137-a542-0ddd73d62a9f

**Comment by pallaviii21** [comment]
okay

**Comment by pallaviii21** [comment]
So what can I do now ?

**Comment by Harshdev098** [comment]
delete the .vs code folder or file that you have commited in this PR

**Comment by Harshdev098** [comment]
Also change the styling of the hamburger

**Comment by pallaviii21** [comment]
sorry for that, I deleted that folder

File: .vscode/settings.json (hunk 1 above)
Comment (pallaviii21) [reply .vscode/settings.json:2]: do I need to delete this then

Comment (Harshdev098) [reply .vscode/settings.json:2]: yes

**Comment by pallaviii21** [comment]
I have made the changes as per you said in the PR. But the .vscode file is still there, I'm sorry for that. Can you please request for change again. I promise ,I won't disappoint you this time.

**Comment by Harshdev098** [comment]
Please make another PR for the issue with the updated changes because there are many unwanted changes in this.
Its not required to update the folder structure, the new files you have made in this gssoc directory feels like duplicate of style.css and index.html 
You can write the styling in the style.css . I am closing this PR for now

Screenshot from 2024-10-07 01-47-14

**Pull request closed without merging**